### PR TITLE
Add a createStore function to disco pane

### LINKED ISSUE #214: Fix missing createStore in disco pane app
The base server assumes a createStore function so as a minimum we need that to get the disco pane server back running correctly.

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #214 

There's no reducers yet but this starts us off with a basic store so that the server doesn't blow up.

### CODE REVIEW

File: src/disco/store.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { applyMiddleware, createStore as _createStore, combineReducers } from 'redux';
+import { reducer as reduxAsyncConnect } from 'redux-async-connect';
+import createLogger from 'redux-logger';
+
+export default function createStore(initialState = {}) {
```
Comment (muffinresearch) [inline src/disco/store.js]: At some point we should look to DRY this up between the apps.

Comment (mstriemer) [inline src/disco/store.js]: I think at some point we'll only want to include `createLogger()` when in development and maybe even only on the client (I find it really noisy but sometimes useful in the console). That might be a good time to figure out a good way of doing so.

Comment (muffinresearch) [inline src/disco/store.js]: I added a comment to #182

**Comment by mstriemer** [comment]
It looks like `src/disco/store.js` isn't being included in the coverage checks. I'm guessing because it doesn't have a test file and we're only looking for files in certain folders [1]. Perhaps we should put the store in `src/disco/store/index.js` just to keep that simple (and add it to `test-runner.js`.

r+wc+coverage

[1] https://github.com/mozilla/addons-frontend/blob/2febfb4568519389cb226deae96e78fad5149a9a/test-runner.js#L5

**Comment by muffinresearch** [comment]
I'll definitely fix the coverage in a bit when there's some more code to cover - initially this is just un-breaking it so I can sort out the code splitting.

Ideally we should improve way coverage happens so it's more automatic. We need to think about a good/better way to do that since we need to exclude the server tests.

**Pull request merged**